src/request_list.js

```diff
@@ -144,6 +144,8 @@ const CONTENT_TYPE_BINARY = 'application/octet-stream';
 /**
  * Represents a static list of URLs to crawl.
  * The URLs can be provided either in code or parsed from a text file hosted on the web.
+ * `RequestList` is used by {@link BasicCrawler}, {@link CheerioCrawler}
+ * and {@link PuppeteerCrawler} as a source of URLs to crawl.
```

Comment:
and PlaywrightCrawler

Reply:
Added PlaywrightCrawler here and in other places through the docs where applicable